events_structs.go

```diff
@@ -760,150 +770,3 @@ type EventTfaDisabledType struct {
-		e = &EventDeleteDomainType{Event: ev}
-	case EventUpgradeDatabase:
-		e = &EventUpgradeDatabaseType{Event: ev}
-	case EventNewAddon:
-		e = &EventNewAddonType{Event: ev}
-	case EventUpgradeAddon:
-		e = &EventUpgradeAddonType{Event: ev}
-	case EventDeleteAddon:
-		e = &EventDeleteAddonType{Event: ev}
-	case EventResumeAddon:
-		e = &EventResumeAddonType{Event: ev}
-	case EventSuspendAddon:
-		e = &EventSuspendAddonType{Event: ev}
-	case EventNewCollaborator:
-		e = &EventNewCollaboratorType{Event: ev}
-	case EventAcceptCollaborator:
-		e = &EventAcceptCollaboratorType{Event: ev}
-	case EventDeleteCollaborator:
-		e = &EventDeleteCollaboratorType{Event: ev}
-	case EventNewVariable:
-		e = &EventNewVariableType{Event: ev}
-	case EventEditVariable:
-		e = &EventEditVariableType{Event: ev}
-	case EventEditVariables:
-		e = &EventEditVariablesType{Event: ev}
-	case EventDeleteVariable:
-		e = &EventDeleteVariableType{Event: ev}
-	case EventAddCredit:
-		e = &EventAddCreditType{Event: ev}
-	case EventAddPaymentMethod:
-		e = &EventAddPaymentMethodType{Event: ev}
-	case EventAddVoucher:
-		e = &EventAddVoucherType{Event: ev}
-	case EventNewKey:
-		e = &EventNewKeyType{Event: ev}
-	case EventEditKey:
-		e = &EventEditKeyType{Event: ev}
-	case EventDeleteKey:
-		e = &EventDeleteKeyType{Event: ev}
-	case EventPaymentAttempt:
-		e = &EventPaymentAttemptType{Event: ev}
-	case EventNewAlert:
-		e = &EventNewAlertType{Event: ev}
-	case EventAlert:
-		e = &EventAlertType{Event: ev}
-	case EventDeleteAlert:
-		e = &EventDeleteAlertType{Event: ev}
-	case EventNewAutoscaler:
-		e = &EventNewAutoscalerType{Event: ev}
-	case EventEditAutoscaler:
-		e = &EventEditAutoscalerType{Event: ev}
-	case EventDeleteAutoscaler:
-		e = &EventDeleteAutoscalerType{Event: ev}
-	case EventAddonUpdated:
-		e = &EventAddonUpdatedType{Event: ev}
-	case EventStartRegionMigration:
-		e = &EventStartRegionMigrationType{Event: ev}
-	case EventNewLogDrain:
-		e = &EventNewLogDrainType{Event: ev}
-	case EventDeleteLogDrain:
-		e = &EventDeleteLogDrainType{Event: ev}
-	case EventNewAddonLogDrain:
-		e = &EventNewAddonLogDrainType{Event: ev}
-	case EventDeleteAddonLogDrain:
-		e = &EventDeleteAddonLogDrainType{Event: ev}
-	case EventNewNotifier:
-		e = &EventNewNotifierType{Event: ev}
-	case EventEditNotifier:
-		e = &EventEditNotifierType{Event: ev}
-	case EventDeleteNotifier:
-		e = &EventDeleteNotifierType{Event: ev}
-	case EventEditHDSContact:
-		e = &EventEditHDSContactType{Event: ev}
-	case EventCreateDataAccessConsent:
-		e = &EventCreateDataAccessConsentType{Event: ev}
-	case EventNewToken:
-		e = &EventNewTokenType{Event: ev}
-	case EventRegenerateToken:
-		e = &EventRegenerateTokenType{Event: ev}
-	case EventDeleteToken:
-		e = &EventDeleteTokenType{Event: ev}
-	case EventTfaEnabled:
-		e = &EventTfaEnabledType{Event: ev}
-	case EventTfaDisabled:
-		e = &EventTfaDisabledType{Event: ev}
-	// Deprecated events. Replaced by equivalent with SCM in the name instead of
-	// Github
```

Comment:
With the Go generator, with lose this information that these are deprecated events. Maybe move this comment near the `EventLinkGithub` and `EventUnlinkGithub` structure definition?

Reply:
It's already present:

```
	// EventLinkGithub and EventUnlinkGithub events are kept for
	// retro-compatibility. They are replaced by SCM events.
	EventLinkGithub   EventTypeName = "link_github"
	EventUnlinkGithub EventTypeName = "unlink_github"
```

Reply:
Oh sorry, I searched for the string "deprecated" in the page 😅